### PR TITLE
Added sampling excluded and included types to host schema

### PR DESCRIPTION
**Pull request by gzuber**
I'm not convinced the Regex is needed or should be included, but I had fun making it 😃

### CODE REVIEW

**Comment by fabiocav** [comment]
Is this still applicable? Was a similar change sent to schema store?

**Comment by gzuber** [comment]
Yes, I believe this is still applicable and needs to be merged. As far as I know it hasn't been reviewed. I recently noticed this waiting and have been trying to bring it up but haven't had a chance to sync up.

**Comment by fabiocav** [comment]
Sounds good. Can you please rebase and request review again? Thanks!

File: schemas/host.json [hunk 1]
```diff
@@ -431,6 +431,16 @@
                       "description": "The target rate that the adaptive algorithm aims for on each instance",
                       "type": "integer",
                       "default": 5
+                    },
+                    "excludedTypes": {
+                      "description": "A semi-colon delimited list of types that should be excluded from sampling. All types not listed will be sampled.",
```
Comment (brettsam) [inline schemas/host.json:436]: In the description, can you list the possible types (same with below)? Types are listed here: https://docs.microsoft.com/en-us/azure/azure-monitor/app/sampling#configuring-adaptive-sampling-for-aspnet-applications

You may be able to just copy their description as-is for these...

**Comment by brettsam** [comment]
FYI -- this change needs to be applied to https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/host.json as well.

**Pull request merged**